File: .sdk/tm/php/utility/PrepareBody.php
<?php
declare(strict_types=1);

// Univec SDK utility: prepare_body

class UnivecPrepareBody
{
    public static function call(UnivecContext $ctx): mixed
    {
        if ($ctx->op->input === 'data') {
            return ($ctx->utility->transform_request)($ctx);
        }
        return null;
    }
}

File: .sdk/tm/csharp/utility/PrepareBody.cs
// Univec SDK utility: prepareBody.

namespace UnivecSdk.Util;

public static partial class SdkUtility
{
    internal static object? PrepareBodyUtil(Context ctx)
    {
        var op = ctx.Op!;

        if (op.Input == "data")
        {
            var body = ctx.Utility!.TransformRequest(ctx);
            return body;
        }

        return null;
    }
}
